Gems/AudioEngineWwise/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+if(NOT COMMAND o3de_gem_setup)
+    # Backwards compatibility for older engines
```

Comment:
Is a backwards compatibility check necessary if this gem is still in the engine repo and not o3de-extras?

Reply:
This particular line is for engines that do not have the `o3de_gem_setup` function.  The development branch has this function but the latest point release doesn't.

Reply:
Yeah, but isn't this PR being merged into development? This line would make sense if this change is going to be cherry picked into the point release I guess

Reply:
I was actually considering bypassing merging it to the point release, and instead getting it included in the next major release, and getting it added to the canonical repo.json so that it is available publicly after a brief period of testing.  Ideally, for the game jam, though that's coming up fast.